### PR TITLE
LPS-153214 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-153214

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Move to FragmentsUpgrade.testcase

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUseCaseUpgrade.testcase [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		SearchAdministration.executeReindex();
+	}
+
+	@description = "This is a test for LPS-153154. View embedded instantiable fragment after upgrade from 7.1.10.5"
+	@priority = "4"
+	test ViewEmbeddedInstantiableFragmentAfterUpgradeFrom71105 {
+		property data.archive.type = "data-archive-fragment-with-instantiable-widget";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property portal.version = "7.1.10.5";
+		property test.assert.warning.exceptions = "true";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUseCaseUpgrade.testcase:25]: Not Recommend, the case should fail prior to run if error happens during upgrade so no need this property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUseCaseUpgrade.testcase [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUseCaseUpgrade.testcase:26]: Add steps to view fragments on view mode first

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you modify according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you modify according to comments? Thanks

Hi @Tim-Cao, PR updated. Would you have a look at? Thanks!

**Comment by Tim-Cao** [comment]
See changes @ozysouza
https://github.com/liferay-echo/liferay-portal/pull/8129/commits/c5c607b2c83175937fb08491457bffe0a6bfdf63

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**